Competition2018/src/test/java/competition/subsystems/drive/commands/RotateToHeadingCommandTest.java

```diff
@@ -0,0 +1,49 @@
+package competition.subsystems.drive.commands;
+
+import static org.junit.Assert.assertEquals;
+import org.junit.Test;
+
+import xbot.common.injection.BaseWPITest;
+import xbot.common.subsystems.pose.BasePoseSubsystem;
+import competition.BaseCompetitionTest;
+import competition.subsystems.drive.commands.RotateToHeadingCommand;
+
+public class RotateToHeadingCommandTest extends BaseCompetitionTest {
+    
+    RotateToHeadingCommand rotate;
+    BasePoseSubsystem pose;
+
+    @Override
+    public void setUp() {
+        super.setUp();
+        rotate = injector.getInstance(RotateToHeadingCommand.class);
+    }
+    
+    @Test
+    public void testTurnLeft() {
+        mockRobotIO.setGyroHeading(0);
+        rotate.setGoalHeading(90);
+        rotate.initialize();
+        rotate.execute();
+        assertEquals(.5, rotate.getRotatePower(), 0.001);
```

Comment:
Instead of getting the command's rotate power, get the power on the drive talons.

Reply:
done